src/bloqade/task/base.py

```diff
@@ -142,6 +151,15 @@ def _filter(
         filter_perfect_filling: bool = True,
         clusters: Union[tuple[int, int], Sequence[Tuple[int, int]]] = [],
     ):
+        """
```

Comment:
this is a private method, there is no need to put docstring on it. As I mentioned we should write docstring for user space methods first.